pandas/plotting/_matplotlib/core.py

```diff
@@ -95,6 +95,8 @@ def __init__(
         ylim=None,
         xticks=None,
         yticks=None,
+        xlabel: Optional[str] = None,
+        ylabel: Optional[str] = None,
```

Comment:
Does this have to be string? Should work with e.g. int / float

Anyway, I tried running this and it looks good! Nice tests

Reply:
yeah, I think it should be string? this is not calling any column of dataframe, but explicitly defined by users who want to give labels for x/y axis. I am not very sure about if users would name it using int/float, quite rare case though

I will later check to see if matplotlib allows to do so or not, if so, maybe we could support too! thanks!

Reply:
I tried running it passing an int, and it worked fine - but yeah, typical case would be to label with a `str`, so it's fine as is, sorry for the noise :)

Reply:
I will change to `Label` then  and add a test case, so that we also support numbers!

Thanks for the feedback!